### ISSUE TITLE
Enable GPU settings on desktop // check default settings

### ISSUE THREAD

**Issue by kauffj** [lbryio/lbry-desktop#5768]
Per a Reddit user [here](https://www.reddit.com/r/lbry/comments/m7tyqz/the_end_of_lbrytv_and_the_future_of_lbry_on_the/gs4inkg/?context=3), Chrome/webkit has settings related to video playback that aren't accessible in our app. We should also examine the default settings and see if any should be changed.

**Comment by tzarebczan** [comment]
Related https://github.com/lbryio/lbry-desktop/issues/4088

**Comment by aquaspy** [comment]
Agreed with this one, hardware acceleration will make a huge difference in performance

**Comment by jessopb** [comment]
I'll look at this after upgrading electron
